### ISSUE TITLE
Request for Model Training Details

### ISSUE THREAD

**Issue by ttthhx** [pwp1208/Domain_Translation_Multi-weather_Restoration#12]
hello,i have difficulties when reproduce the trainning code,i noticed that the weights of loss function (\lambda) is not clarified in your paper,I would appreciate it if you could offer the weights or training code.this is my e-mail:thx1256901488@gmail.com
